test/react.test.js

```diff
@@ -29,8 +29,7 @@ describe('styled', () => {
   })
 
   test('basic render with object as style', () => {
-    const fontSize = 20
-    const H1 = styled.h1({ fontSize })
+    const H1 = styled.h1({ fontSize: 20 })
```

Comment:
We have to support those. Are the array values what are tripping you up? If so I'd rather not support those and do this.